docs/examples/driver_examples/Qcodes example with keithley 2450.ipynb

```diff
@@ -0,0 +1,457 @@
+     "text": [
+      "'keithley.source.function()=current'Please run 'keithley.source.function(voltage)' and try again\n"
+     ]
+    }
+   ],
+   "source": [
+    "try: \n",
+    "    keithley.source.voltage()\n",
+    "except ParameterError as err: \n",
+    "    print(err)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This goes for both the source and sense subsystems "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'keithley.sense.function()=voltage'Please run 'keithley.sense.function(current)' and try again\n"
+     ]
+    }
+   ],
+   "source": [
+    "try: \n",
+    "    voltage = keithley.sense.current()\n",
+    "except ParameterError as err: \n",
+    "    print(err)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We also need to make sure the output is enabled for use the measure (or 'sense') a current or voltage "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "'keithley.output()=False'Please run 'keithley.output(True)' and try again\n"
+     ]
+    }
+   ],
+   "source": [
+    "try: \n",
+    "    voltage = keithley.sense.voltage()\n",
+    "except ParameterError as err: \n",
+    "    print(err)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sweeping measurements "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The instrument has a build-in sweep system. For the first measurement, we drive a current through the resistor and measure the voltage accross it. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "keithley.source.function(\"current\")\n",
+    "keithley.source.auto_range(True)\n",
+    "keithley.source.limit(0.2)\n",
+    "\n",
+    "keithley.sense.function(\"voltage\")\n",
+    "keithley.sense.auto_range(True)\n",
+    "\n",
+    "keithley.sense.four_wire_measurement(True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "initialise_database()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Sweep the current from 0 to 1uA in 10 steps"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 29,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def setup(start, stop, step): \n",
+    "    keithley.sweep.setup(start, stop, step)\n",
+    "    keithley.output(True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def cleanup(): \n",
+    "    keithley.output(True)"
```

Comment:
I assume this should disable the output?

Reply:
Yes, this is wrong